test/bench/modules.js

```diff
@@ -18,36 +18,24 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-/* eslint-disable no-console, no-invalid-this */
-import {Bench} from '@probe.gl/bench';
+// Override aliases to point to publicly accessible github
+// TODO maybe setPathPrefix is enough?
+import ALIASES from '../../test/aliases';
+import {_addAliases} from '@loaders.gl/loader-utils';
 
+import imageBench from '@loaders.gl/images/test/images.bench';
 import coreBench from '@loaders.gl/core/test/core.bench';
 import csvBench from '@loaders.gl/csv/test/csv.bench';
 import jsonBench from '@loaders.gl/json/test/json-loader.bench';
-import dracoBench from '@loaders.gl/draco/test/draco.bench';
-import imageBench from '@loaders.gl/images/test/images.bench';
+// import dracoBench from '@loaders.gl/draco/test/draco.bench';
```

Comment:
`dracoBench` doesn't work?

Reply:
It works if you run the example separately, it doesn't work in the gatsby website. Module loading fails. 

I have spent too much time debugging how to make loader benchmarks work in gatsby to fix it right now :)